EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -385,7 +385,7 @@ num_threads = min(10, Threads.nthreads())
 # ╔═╡ 88b43e23-1e06-4716-b284-76e8afc6171b
 inference_method = EpiMethod(
     pre_sampler_steps = [ManyPathfinder(nruns = 4, maxiters = 100)],
-    sampler = NUTSampler(adtype = AutoReverseDiff(true),
+    sampler = NUTSampler(adtype = AutoReverseDiff(),
```

Comment:
This is a change from eager to lazy compilation? Trying to follow what's happening and the ADTypes 404'd `AutoReverseDiff()`

Reply:
The docs are [here](https://sciml.github.io/ADTypes.jl/stable/#ADTypes.AutoReverseDiff). The new API is default no tape compilation. 

This **is** a change from the getting started example I now note, but I think having the getting_started example as more likely to throw an error rather than get stuck on a branch (the silent risk of tape compiled reverse AD in all langs) should have been the default anyway.